trojsten/polls/models.py

```diff
@@ -0,0 +1,31 @@
+from datetime import datetime
+
+from django.conf import settings
+from django.db import models
+from django.utils import timezone
+
+
+class Question(models.Model):
+    text = models.CharField(max_length=1000)
+    created_date = models.DateTimeField(default=timezone.now)
+    deadline = models.DateTimeField(default=datetime(2020, 12, 31, 23, 59, 59))
+
+    def __str__(self):
+        return self.text
+
+
+class Answer(models.Model):
+    text = models.CharField(max_length=1000)
+    question = models.ForeignKey("Question", on_delete=models.CASCADE)
+
+    def __str__(self):
+        return f'Answer "{self.text}" to question "{str(self.question)}"'
+
+
+class Vote(models.Model):

```

Comment:
Neviem ake su realne poziadavky na anonymitu, ale toto anonymne nie je.
Ak to nevadi tak je to takto ok. Ak treba anonymne, tak k uzivatelovi by sa malo len zapisat ci v ankete hlasoval a hlas by sa mal zapisat k odpovedi anonymne.

Reply:
Anonymitu vyriesime tak, ze len uzka skupina veducich, superadmini, budu vidiet v admin sekcii hlasy, ostatny ich vidiet nebudu.